website/docs/r/stack_hci_deployment_setting.html.markdown

```diff
@@ -0,0 +1,592 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_stack_hci_deployment_setting"
+description: |-
+  Manages a Stack HCI Deployment Setting.
+---
+
+# azurerm_stack_hci_deployment_setting
+
+Manages a Stack HCI Deployment Setting.
```

Comment:
Can we also get a note added about the additional resources created on behalf of the user in their resource group to help ensure they are aware of their existence and importance, and that Terraform will attempt to remove these on deletion or recreation of this resource?